XcodeServerSDK/XcodeServer.swift

```diff
@@ -322,6 +322,38 @@ public extension XcodeServer {
         }
     }
     
+    /**
+    XCS API call for retrievieng specified Integration.
+    
+    - parameter integrationId: ID of integration which is about to be retrieved.
+    - parameter completion:
+    - Optional retrieved integration.
+    - Optional operation error.
+    */
+    public func retrieveIntegration(integrationId: String, completion: (integration: Integration?, error: NSError?) -> ()) {
```

Comment:
I'm not insistent on keeping it this way, but so far I've tried to name most of these API calls with words starting with `get`, `post` etc, to sort of mirror the HTTP verb convention. There are exceptions, like `createBot` (which could just be `postBot`), but I think, for the sake of users' autocompletion, that we should keep the first words as predictable as possible. Here I'd name it just `getIntegrations`. Whadaya think?